src/jit/codegenlegacy.cpp

```diff
@@ -22319,7 +22319,7 @@ regNumber          CodeGen::genPInvokeCallProlog(LclVarDsc*            frameList
         regTracker.rsTrackRegTrash(tcbReg);
     }
 
-#ifdef _TARGET_X86_
+#if defined(_TARGET_X86_) || defined(_TARGET_ARM_)
     /* mov   dword ptr [frame.callSiteTracker], esp */
```

Comment:
Without this modification, I get Illegal Instruction that appears to be caused by genPInvokeCall (comparing `REG_OPT_RSVD` (loaded from `frame.callSiteTracker`) with `esp` and send it to `BREAKPOINT` (SIGILL) if not matched. The comparison code is also activated for _ARM_; thus, we need either to enable this code or to disable the comparison code.

Anyway, this code works so far.